crates/maybenot-ffi/cbindgen.toml

```diff
@@ -0,0 +1,19 @@
+# See https://github.com/mozilla/cbindgen/blob/master/docs.md#cbindgentoml
+
+language = "C"
+
+# header = "/* Text to put at the beginning of the generated file. Probably a license. */"
+# trailer = "/* Text to put at the end of the generated file */"
+autogen_warning = "/* Warning, this file is autogenerated by cbindgen. Don't modify this manually. */"
+include_version = true
+
+braces = "SameLine"
```

Comment:
I suggest not specifying all options that are the same as the default anyway. Mostly redundant. And if the cbindgen ever changes their default there is probably a fairly good reason for doing so. Unless we somehow technically depend on a config option being a certain value, then we should be explicit. But if it's just style, then not much need to deviate from the defaults IMHO.

Reply:
Agree, some of these are not the defaults, but I don't remember why I decided to keep the rest of them :thinking:

Reply:
I removed the options that were the same as the default